app/src/main/res/layout/fragment_tag_list.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".fragments.TagListFragment"
```

Comment:
We'll want to update to be fragment_tag_recycler_view and change the tools:context=".fragments.TagListFragment" to tools:context=".fragments.TagRecyclerViewFragment"